Sources/MapboxDirections/RouteStep.swift

```diff
@@ -519,7 +519,7 @@ open class RouteStep: Codable {
         try maneuver.encode(instructions, forKey: .instruction)
         try maneuver.encode(maneuverType, forKey: .type)
         try maneuver.encodeIfPresent(maneuverDirection, forKey: .direction)
-        try maneuver.encodeIfPresent(maneuverLocation, forKey: .location)
+        try maneuver.encode(CLLocationCoordinate2DCodable(maneuverLocation), forKey: .location)
```

Comment:
Are there any maneuver types for which there’s no `maneuverLocation` present?

Reply:
As I understand what `Maneuver` is, it always has a location where it "occurs" or "vocalized". Also, in the code, `maneuverLocation` is non-nil.